app/dao/uploads_dao.py

```diff
@@ -10,6 +10,25 @@
 from app.utils import midnight_n_days_ago
 
 
+def _get_printing_day(created_at):
+    return func.date_trunc(
+        'day',
+        func.timezone('Europe/London', func.timezone('UTC', created_at)) + text(
+            "interval '6 hours 30 minutes'"
```

Comment:
Date functions are always confusing and over complicated. Can you add a comment to explain what's happening and why the interval text is needed, please?

Reply:
Comments added in c61f7e70